conda-requirements.txt

```diff
@@ -36,4 +35,4 @@ python-stratify
 pyugrid
 
 # Iris extensions (i.e. key tools that depend on Iris)
-# iris_grib
+iris_grib>=0.12
```

Comment:
So I thought this was commented out for a reason!? iris extensions shouldn't be installed by iris because *they themselves* depend on Iris.